src/cljs/harja/tiedot/urakka/yllapitokohteet/muut_kustannukset.cljs

```diff
@@ -0,0 +1,95 @@
+(ns harja.tiedot.urakka.yllapitokohteet.muut-kustannukset
+  "Päällystyskohteiden Muut kustannukset-taulukon tiedot.
+
+  Tässä taulukossa näytetään sanktiot jotka eivät liity mihinkään ylläpitokohteeseen,
+  sekä käsin syötetyt vapaamuotoiset muut kustannukset. Sanktioita ei voi muokata tai lisätä.
+
+  Nämä kustannukset lasketaan mukaan kustannusyhteenvetotaulukkoon käyttäen kohteet-reaktiota."
+  (:require
+   [reagent.core :refer [atom] :as r]
+   [harja.loki :refer [log tarkkaile!]]
+   [harja.tiedot.urakka :as tiedot-urakka]
+   [harja.tiedot.urakka.laadunseuranta.sanktiot :as tiedot-sanktiot]
+   [harja.tiedot.navigaatio :as nav]
+
+   [cljs.core.async :refer [<! pipe chan]]
+   [harja.asiakas.kommunikaatio :as k]
+   [harja.pvm :as pvm]
+   [clojure.string :as s]
+   [clojure.set :refer [rename-keys]])
+  (:require-macros [reagent.ratom :refer [reaction]]
+                   [harja.atom :refer [reaction<!]]
+                   [cljs.core.async.macros :refer [go]]))
+
+(defonce nakymassa? (atom nil)) ;; komp/lippu päivittää tätä
+(defonce riveja-tallennettu-laskuri (atom 0)) ;; käytetään potkaisemaan hakureaktiota tallennuksen jälkeen
```

Comment:
Tää kuulostaa vähän oudolta? Meillä on harja.atom/paivita! joka pakottaa reaktion päivittymään. Olisiko tämä parempi kuin käyttää näitä omia "potkaisuja"? :)